deploy/test/test_cases/TEST_ID_17_helm_job_deploys_successfully.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This test verifies that the Secrets Provider Job deploys successfully and Conjur secret appears in pod
+# TODO verify Job start up and completed logs
+
+function main() {
+  setup_helm_environment
+
+  deploy_chart
+
+  validate_job_completion
+
+  # Deploy app for Helm testing
+  deploy_helm_app
+
+  pod_name=$($cli_with_timeout get pods --namespace=$APP_NAMESPACE_NAME --selector app=test-env --no-headers | awk '{print $1}' )
+  verify_secret_value_in_pod $pod_name "TEST_SECRET" "supersecret"
+}
+
+function deploy_chart {
```

Comment:
I prefer not having function in the test it self. We can use those functions in other tests as well now or in the future.
For example you always run fill_helm_chart and helm install maybe with different parameters but i prefer this file will be clean.
Moving from clean files to BDD in the future will be easier as well.